stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -1231,7 +1237,8 @@ extension NSSet {
   ///   receiver.
   @objc(_swiftInitWithSet_NSSet:)
   public convenience init(set anSet: NSSet) {
```

Comment:
I suppose `copy()` covers the use case of getting a known-immutable instance just as well, but yeah, we can discuss removing these initializers separately.